docker-compose.yml

```diff
@@ -142,6 +142,7 @@ services:
       SUBDOMAIN_OFFSET: 0
       USE_SECURE_SESSION_COOKIES: 'false'
       PDF_SERVICE_API_URL: 'http://pdf-service-api:3000'
+      LOGGER_LEVEL: 'debug'
```

Comment:
Is this needed?

Reply:
Yes, otherwise the messages are not logged in local